ansible/library/interface_facts.py

```diff
@@ -313,21 +313,18 @@ def main():
     cmd_prefix = ''
     cmd = '/usr/bin/python {}'.format(INTF_IP_GET_INFO_SCRIPT)
 
-    for namespace in multi_asic.get_front_end_namespaces():
-        if namespace_passed and namespace != namespace_passed:
-            continue
-        # If the user passed a namespace parameter invoke that script with the cmd_prefix
-        if namespace:
-            cmd_prefix = 'sudo ip netns exec {} '.format(namespace)
-        rc, output, err = module.run_command(cmd_prefix + cmd, use_unsafe_shell=True)
-        if rc != 0:
-            module.fail_json(msg="Failed to run {}, rc={}, stdout={}, stderr={}".format(cmd, rc, output, err))
-
-        # Get the output from the gather interface info script.
-        if output:
-            ips_interfaces = json.loads(output)
-            interfaces.update(ips_interfaces["interfaces"])
-            ips.update(ips_interfaces["ips"])
+    # If the user passed a namespace parameter invoke that script with the cmd_prefix
+    if namespace_passed:
+        cmd_prefix = 'sudo ip netns exec {} '.format(namespace_passed)
```

Comment:
@SuvarnaMeenakshi  why we remove for loop. This will breal acl test case which does not passs namespace and we get for all frontend interface.

Reply:
Fixed, added loop to go through all namespaces and not just front end namespace so that interface_facts is able to retrieve for all namespace.